css/stylesheet.css

```diff
@@ -193,9 +202,37 @@ blockquote {
   margin-top: 0;
 }
 
+ol {
+  margin: 30px auto;
+  border: 3px solid white;
+  width: 55%;
+  padding: 30px 50px;
+  font-size: 20px;
+  border-radius: 15px;
+}
+
+ol a {
+  color: #702F8A;
+  text-decoration: underline;
+}
+
 
 @media (max-width: 768px) {
 
+  /*.menuIcon {
+    display: inline-block;
+  }*/
+
+  nav ul {
+    float: none;
+    display: none;
```

Comment:
Does this work? I would expect hiding the `<ul>` to hide its children elements (the `<li>`s) even if you set those to `display: block`. Either way it's a little bit odd in the CSS to read. The `<ul>` doesn't have any natural styling so it's safe to leave as display: block. Removing the float is right though.